client/theme/theme.js

```diff
@@ -28,7 +28,7 @@ const theme = createMuiTheme({
       light: colorTextSecondaryLight,
     },
     background: {
-      default: '#1A1A1A',
+      default: '#0f181f',
```

Comment:
It looks like we generally use uppercase for color codes, so let's stick with that for consistency.